gulpfile.js

```diff
@@ -85,14 +85,14 @@ gulp.task('copy-app-common', function() {
     gulp.src('./node_modules/whatwg-fetch/fetch.js')
       .pipe(rename('js/polyfills/fetch.js'))
   )
-  .pipe(gulp.dest(DIST_APP_ROOT));
+    .pipe(gulp.dest(DIST_APP_ROOT));
```

Comment:
These alignements were not all consistent, so WebStorms code formatting put them all back together.